pages/common/packtpub.md

```diff
@@ -0,0 +1,32 @@
+# packtpub
+
+> Download freely offered books from packtpub.com.
+> More information: <https://github.com/vladimyr/packtpub-cli>.
+
+- Download the daily offer book into the current directory with the specified book format (defaults to `pdf`):
+
+`packtpub download --type {{pdf|ebup|mobi}}`
+
+- Download the daily offer book into the specified directory:
+
+`packtpub download --dir {{path/to/directory}}`
+
+- Start an interactive login to packtpub.com:
```

Comment:
What about? (not sure if "good english"). It seems obvious you log in to `packtpub.com`. 😇
```suggestion
- Log in interactively:
```

Reply:
Yeah, it is now, but what about a user who has the command installed (perhaps he automated the installation of many) and it now seeing what it is.  If he know about packtpub.com he'll know what the command's for, but if he's not heard of the site before he might wonder what it logs in to.

Do you see my logic behind this wording?